src/languages/marko.coffee

```diff
@@ -1,8 +1,15 @@
+# Get Atom defaults
+scope = ['text.marko']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 4)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
 module.exports = {
 
   name: "Marko"
   namespace: "marko"
-  fallback: ['html']
```

Comment:
I see...my understanding was wrong. Thanks for clarifying! I have added `html` back as a fallback. Do you have any other concerns or see any other outstanding issues? Thanks, again, for all your help and maintaining this project.